test/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -80,8 +80,25 @@ public static UnitType carrier(final GameData data) {
     return unitType(Constants.UNIT_TYPE_CARRIER, data);
   }
 
+  // Some units hard coded here rather than placed in Constants at the insistence of community members Ref: Pull Request 1074
   public static UnitType tacBomber(final GameData data) {
-    return unitType(Constants.UNIT_TYPE_TAC_BOMBER, data);
+    return unitType("tactical_bomber", data);			
+  }
+
+  public static UnitType airbase(final GameData data) {
+    return unitType("airbase", data);			
+  }
+
+  public static UnitType harbour(final GameData data) {
+    return unitType("harbour", data);			
+  }
+
+  public static UnitType factory_major(final GameData data) {
```

Comment:
first character should always be lower case. 
For more details, see: https://google.github.io/styleguide/javaguide.html#s5-naming

We agreed to go by the google guidelines by default, with one or two exceptions requested by Veqryn (line length does not have to be 80, 120 I think is the limit set in the formatter, can't recall what the other exception was, was minor.)